api-put-bucket.go

```diff
@@ -163,7 +163,8 @@ func (c Client) SetBucketPolicy(bucketName string, objectPrefix string, bucketPo
 	}
 
 	policyInfo, err := c.getBucketPolicy(bucketName)
```

Comment:
if all callers to `getBucketPolicy` should check for "NoSuchBucketPolicy", can't we check for the corresponding `errResponse.Code` inside the function and return `sentinelBucketAccessPolicy, nil`?